users/tests/test_user_model.py

```diff
@@ -280,32 +274,40 @@ def test_deactivate_user_sets_fields_correctly(generated_password: str) -> None:
 
 
 @pytest.mark.django_db()
-def test_anonymize_user_successfully(generated_password: str) -> None:
-    user = User.objects.create(
-        nickname="test_user",
-        email="test@example.com",
-        password=generated_password,
-        is_active=False,
-        deactivated_at=timezone.now(),
-        reactivate_until=timezone.now(),
-    )
+def test_anonymize_active_user_does_not_work(user: User) -> None:
```

Comment:
Please update the name to explain why this user was not anonymized. Something like
test_cant_anonymize_active_user or sth.